core/runtime/core_builtin.odin

```diff
@@ -591,7 +591,9 @@ assign_at_elems :: proc(array: ^$T/[dynamic]$E, index: int, args: ..E, loc := #c
 		copy(array[index:], args)
 		ok = true
 	} else {
-		resize(array, index+1+len(args), loc) or_return
+		new_size := len(array)+len(args)
+		if len(array) == index && len(args) == 0 do new_size += 1
```

Comment:
We don't allow the usage of `do` in core/vendor because the `-disallow-do` flag exists and we don't want core/vendor code to come up with it enabled.
```suggestion
		if len(array) == index && len(args) == 0 { new_size += 1 }
```

Reply:
I applied your suggestions